docs/dev/reference/glossary/origin-frame.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Origin frame
+id: origin-frame
+full_link:
+short_description: The origin frame is the frame at the base of an arm or other component that has a complex kinematics chain.
+---
+
+The origin frame is the frame at the base of an arm, gantry, or other component that has a complex kinematics chain.
```

Comment:
All robot components have an origin frame if they have a frame, regardless of "complex kinematics chain" or not

Reply:
```suggestion
The origin frame is the frame at the base of an arm, gantry, or other component.
```